tests/framework/clients/rancher/v1/client.go

```diff
@@ -283,6 +287,45 @@ func (c *NamespacedSteveClient) Update(existing *SteveAPIObject, updates any) (*
 	return c.SteveClient.Update(existing, updates)
 }
 
+func (c *NamespacedSteveClient) PerformPutCaptureHeaders(host, token, endpoint, name string, payload interface{}) (http.Header, []byte, error) {
+	body, err := yaml.Marshal(payload)
+	if err != nil {
+		return nil, nil, fmt.Errorf("error marshalling payload: %v", err)
+	}
+
+	url := fmt.Sprintf("https://%v/v1/%v/%v/%v", host, endpoint, c.namespace, name)
```

Comment:
since we know the Steve type ie endpoint, we don't need to pass that as a parameter either.

Reply:
updated, thank you! 😁